R/derive_locf_records.R

```diff
@@ -75,20 +99,11 @@
 #'   "CDISC01", "01-701-1015", "DIABP",        2,    NA,        4, "WEEK 4",
```

Comment:
Could you add `VSSEQ` to `advs`? Then the results for `imputation = "add"` and `imputation = "update_add"` will differ.

If this requires too much space, you could remove `STUDYID` or use a shorter value for `USUBJID'`.